### PR TITLE
 Implemented Make model

### PR DESCRIPTION
**Pull request by Pravallika2812**

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -25,9 +25,20 @@
 Returns: None
 '''
 def makeModel(data):
+    data["rows"]=10
+    data["cols"]=10
+    data["boardsize"]=500
+    data["cellsize"]=50
+    data["numships"]=5
+    data["Computerboard"]=[[]]
+    data["Userboard"]=[[]]
```
Comment (Pragya28) [inline battleship.py:34]: These

File: battleship.py [hunk 2]
```diff
@@ -25,9 +25,20 @@
 Returns: None
 '''
 def makeModel(data):
+    data["rows"]=10
+    data["cols"]=10
+    data["boardsize"]=500
+    data["cellsize"]=50
```
Comment (Pragya28) [inline battleship.py:31]: Calculate cell size using board size and number of rows/cols. Don't write the value directly.

**Review by Pragya28 — CHANGES_REQUESTED** [review]

File: battleship.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Pravallika2812) [reply battleship.py]: Modified Makemodel function Mam

File: battleship.py (hunk 2 above)
Comment (Pravallika2812) [reply battleship.py]: Modified Make Model Function

**Review by Pragya28 — COMMENTED** [review]
Good work

**Pull request merged**